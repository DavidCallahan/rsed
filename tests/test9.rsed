#!/home/dcallahan/bin/rsed
foreach past "^power management"
   if "^(.*\S)\s*:(.*)" then
      set $name="cpuinfo_" replace(" ", "_", $1)
      set $value=trim($2)
      print $name "=\"" $value "\"" 
      skip 
   else 
      error unexpected line ":" $CURRENT
   end
end


