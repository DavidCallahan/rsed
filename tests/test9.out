cpuinfo_processor="0"
cpuinfo_vendor_id="GenuineIntel"
cpuinfo_cpu_family="6"
cpuinfo_model="45"
cpuinfo_model_name="Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz"
cpuinfo_stepping="6"
cpuinfo_microcode="0x615"
cpuinfo_cpu_MHz="2200.000"
cpuinfo_cache_size="20480 KB"
cpuinfo_physical_id="0"
cpuinfo_siblings="16"
cpuinfo_core_id="0"
cpuinfo_cpu_cores="8"
cpuinfo_apicid="0"
cpuinfo_initial_apicid="0"
cpuinfo_fpu="yes"
cpuinfo_fpu_exception="yes"
cpuinfo_cpuid_level="13"
cpuinfo_wp="yes"
cpuinfo_flags="fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf eagerfpu pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic popcnt tsc_deadline_timer aes xsave avx lahf_lm arat epb pln pts dtherm tpr_shadow vnmi flexpriority ept vpid xsaveopt"
cpuinfo_bugs=""
cpuinfo_bogomips="4399.72"
cpuinfo_clflush_size="64"
cpuinfo_cache_alignment="64"
cpuinfo_address_sizes="46 bits physical, 48 bits virtual"
cpuinfo_power_management=""
